CHANGELOG.md

```diff
@@ -0,0 +1,29 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format follows [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [0.6.2] - 2020-06-08
+
+### Added
+
+-   Added support for downloading plugins from platform specific URLs. 
+    For example on Mac, Terrawrap will try downloading from `<plugin_url>/Darwin/x86_64` first and fallback to 
+    downloading from `<plugin url>`. See README for more details.
+    
+-   Added support for downloading from plugins from S3 using the AWS SDK
+
+## [0.6.1] - 2020-06-08
+
+### Changed
+
+-   Changed `backend_check` to ignore directories that don't contain `.tf` files
+
+## [0.6.0] - 2020-06-05
+
+### Added
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-duplicate-headings] Do not use headings with similar content (10:1)](https://app.codacy.com/gh/amplify-education/terrawrap/pullRequest?prid=5656512)